packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -42,25 +45,24 @@ export class VaRadioOption {
   })
   radioOptionSelected: EventEmitter;
 
-  private handleChange(): void {
+  private handleChange = (): void => {
     this.radioOptionSelected.emit();
-  }
+  };
 
   render() {
-    const ariaDescribedby = (this.checked && this.ariaDescribedby) || null;
+    const { checked, handleChange, label, name, value } = this;
+
     return (
-      <Host
-        aria-checked={this.checked ? `${this.checked}` : 'false'}
```

Comment:
We shouldn't need [aria-checked](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked) now that we have a radio input, but I'll check with Sarah.